src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -46,7 +46,7 @@ def get_address_id_maps(
     if column_name != "address_id":
         raise ValueError(f"Expected `address_id`, got `{column_name}`")
     maps = dict()
-    output_cols_superset = [column_name, "state", "puma"]
+    output_cols_superset = [column_name, "state_id", "state", "puma"]
```

Comment:
Do you need state at all?

Reply:
We get the state abbreviation when we read in the raw results (which is the "state" column) and this is used for selecting cities.

Reply:
But raw_results is now state_id, no? You're talking raw_results as in observer data, right?

Reply:
Yes but we do some cleaning when we initially read the data into memory for results processing.  State_id is in raw_results and when we read it in we create state which is the state abbreviation we want in final outputs.  I could do that same mapping in here to go from the state_id to the abbreviation but it has already been done so it doesn't hurt to include the extra column.

Reply:
The cleaning I am referring to is taking place in load_data in make_results.py and the actual code is in formatter.py

Reply:
ah got it - that roundabout confused me